src/Name/Generator/ControllerNameGenerator.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Jaeger\Symfony\Name\Generator;
+
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpKernel\Event\RequestEvent;
+use Symfony\Component\HttpKernel\Event\TerminateEvent;
+
+class ControllerNameGenerator implements NameGeneratorInterface, EventSubscriberInterface
+{
+    private string $controller;
+
+    public static function getSubscribedEvents(): array
+    {
+        return [RequestEvent::class => ['onRequest', 30], TerminateEvent::class => ['onTerminate', -16384],];
+    }
+
+    public function onRequest(RequestEvent $event): void
+    {
+        $this->controller = $event->getRequest()->attributes->get('_controller', '');
```

Comment:
```suggestion
        $this->controller = (string)$event->getRequest()->attributes->get('_controller', '');
```

I know, it looks like nonsense, but in theory, an attribute can have another type.
Let's cast it explicitly.